products/msexchange.md

```diff
@@ -12,6 +12,15 @@ identifiers:
 -   cpe: cpe:2.3:a:microsoft:exchange_server
 
 releases:
+-   releaseCycle: "subscription"
+    releaseLabel: "Subscription Edition"
+    releaseDate: 2025-07-01
```

Comment:
are you sure this one is the real first release date for this subscription edition ?

Reply:
Check the blog article I linked in issue description:
> Exchange Server Subscription Edition (SE) is now available
> The_Exchange_Team Platinum Contributor
> Jul 01, 2025
> We are excited to announce the general availability of Exchange Server Subscription Edition (SE).